am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessGrant.java

```diff
@@ -13,7 +13,7 @@
 @Builder
 public final class ExplicitAccessGrant {
     private final String resourceId;
-    private final String accessorId;
+    private final Set<String> accessorIds;
```

Comment:
In my mind validation will be done in the following way (just FYI)

```
private final @NotEmpty Set<@NotBlank String> accessorIds;
```

Reply:
👍